packages/wrangler/src/__tests__/upgrade.test.ts

```diff
@@ -0,0 +1,207 @@
+import { readFileSync, writeFileSync } from "fs";
```

Comment:
It would be nice to see some more thorough testing of yarn, npm, and pnpm (maybe with `describe.each`?)

Reply:
I used the different locks in different tests, I can make an explicit test however.

Reply:
If possible, could you run all the tests with each package manager (with `describe.each`)?